### ISSUE TITLE
"Lost entities" below terrain level

### ISSUE THREAD

**Issue by C0mbie** [TheGameCreators/GameGuruRepo#343]
If you try to move an entity in the editor you could accidentally push it far below the terrain level.

If that happens it is really hard to notice such entity as the yellow box indicator disappears if the object is too far away.

Yellow box when the object is close:
![screen4](https://user-images.githubusercontent.com/42276000/43966555-837bd56e-9cc2-11e8-823e-d104afe24ed8.PNG)

I would be nice to improve this feature to show all entities hidden by the terrain at any distance in order to avoid having useless "waste" around the map that cannot be easily recovered an thus possibly improving memory/game performance.

**Comment by AmenMoses** [comment]
Drag select using the mouse over the terrain will highlight any items below the terrain.

**Comment by C0mbie** [comment]
That works but sometimes you don't notice the entities you "lost" below the terrain.

I agree it's not an urgent feature though, but given the fact that you can accidentally lose an entity while trying to set it at the right height, it would be useful to clean up the map.

**Comment by Teabone3** [comment]
An ability to find entities would solve this. Even an ability to hide/show terrain/water within the editor.

**Comment by michaeloldman** [comment]
Would be nice to be able to click an object in the Entity panel, and then use Tab to cycle through each clone of that object in the level.

Reliquia....

**Comment by davetgc** [comment]
I agree, I would also like to see a search for object by name within the level and to cycle through all models that are the same as the current selection.

**Comment by OldFlak** [comment]
This feature has been added 
![image](https://user-images.githubusercontent.com/12246318/154260430-115ed481-db73-40cf-923a-26d2fbc0b714.png)
